src/validations/auth.validations.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable quotes */
+import Response from '../helpers/response';
```

Comment:
Why did you disable quotes?

Reply:
Because the split method can't work with eslint single quote.